### DISCUSSION TITLE
[Question][Hw6] Autograder failure right before deadline(1 day late)

### DISCUSSION THREAD

**Discussion by ashnvael** [prosyslab-classroom/cs348-information-security#727] [Q&A]
Name: Nikolai Kurlovich (20190880)

So unfortunately I was making modifications to my fuzzer almost until the very midnight and in the end just submitted one of the previous versions which had scored 77.8. And unfortunately, it takes quite some time for the autograder to run and I knew the score only after the midnight. I really don't know why it failed this one final time, since the file is completely identical to the submission which scored 77.8. 

![image](https://github.com/prosyslab-classroom/cs348-information-security/assets/96183185/e9297cfb-c7c1-4e7f-a0fc-7b2d1b99f53b)

![image](https://github.com/prosyslab-classroom/cs348-information-security/assets/96183185/1bc81fd8-7b78-4a81-98ed-39dacc1924ad)

Please. I would be very grateful if you could consider my last non-zero submission(75.06), or that 77.8 submission since my last submission is identical to it. I was extremely unlucky that the autograder gave this error on my last, final submissionㅠㅠㅠ. 

Thank you.

**Comment by yeonhee-ryou** [comment]
I regraded your last submission and it will be used

**Comment by ashnvael** [comment]
Thank you~~
